.github/workflows/cwf-integ-test.yml

```diff
@@ -81,12 +86,14 @@ jobs:
           do
             gzip -d $IMAGES
           done
-          docker image load --input cwf_gateway_sessiond.tar
-          docker image load --input cwf_nginx.tar
-          docker image load --input cwf_gateway_python.tar
-          docker image load --input cwf_cwag_go.tar
-          docker image load --input cwf_gateway_go.tar
-          docker image load --input cwf_gateway_pipelined.tar
+          # docker image load --input cwf_gateway_sessiond.tar
```

Comment:
Can you delete instead of commenting, we could always retrieve it from the history :)

Reply:
yes i'm just testing this still :)